src/components/UsersInfo/UsersInfo.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { sendRequest } from '../../sendRequest/sendRequest';
+import { getUsers } from '../../sendRequest/apis';
+import { withRouter } from 'react-router-dom';
+import './UsersInfo.css';
+
+class UsersInfo extends React.Component {
```

Comment:
Ya this is the older version of writing components take a look at the latest react documentation for hooks